src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts

```diff
@@ -21,12 +21,27 @@ export class AddFilterWizardComponent implements OnInit {
   public isValidName = true;
   public payload: any;
   public selectedPluginDescription = '';
-
+  public pluginData = [];
   public filesToUpload = [];
+  public installNewPlugin = false;
+  private REQUEST_TIMEOUT_INTERVAL = 2000;
```

Comment:
for install plugin, please make it 5sec. and  test with expression filter (on RPi) and GUI on another machine

Reply:
I have already tested it on RPi and GUI on another machine.